neofetch

```diff
@@ -111,6 +111,21 @@ get_distro() {
                 # Workarounds for distros that go against the os-release standard.
                 [[ -z "${distro// }" ]] && distro="$(awk '/BLAG/ {print $1; exit}' /etc/*ease /usr/lib/*ease)"
                 [[ -z "${distro// }" ]] && distro="$(awk -F'=' '{print $2; exit}' /etc/*ease /usr/lib/*ease)"
+
+            else
+                distro="$(cat /etc/*-release)" # UUOC, but using usual bash $(< *-file) returns "ambiguous redirect"
```

Comment:
Yup, you can't use `<` on multiple files.

Does this work? (I'm not on my Linux machine right now)

```sh
release_files=(/etc/*-release)
distro="$(< "${release_files[@]}")"
```